app/src/main/java/in/co/rajkumaar/amritarepo/aums/activities/LoginActivity.java

```diff
@@ -81,16 +83,33 @@ public boolean onTouch(View v, MotionEvent event) {
         dialog = new ProgressDialog(this);
         dialog.setCancelable(false);
 
-        SharedPreferences pref = getSharedPreferences("user", Context.MODE_PRIVATE);
+        final SharedPreferences pref = getSharedPreferences("user", Context.MODE_PRIVATE);
+        Encryption enc = null;
+        String rmusername = null;
+        String rmpassword = null;
+        try {
+            enc = new Encryption(LoginActivity.this, "user");
```

Comment:
```suggestion
          final Encryption  enc = new Encryption(LoginActivity.this, "user");
```

Reply:
I need an encryption object inside login button click listener, if i assign final Encryption enc inside try, it won't be visible inside the listener.

Reply:
Done